### PR TITLE
[Sheets] [Pt. 9] Normalize RNGH pressable, use in remaining places

### PR DESCRIPTION
**Pull request by haileyok**
- **remove**
- **close all dialogs**
- **remove a bunch of to-be-useless stuff**
- **remove `Dialog.Handle` uses**
- **rm handle component**
- **type**
- **get scrollview working**
- **tidy**
- **bump**
- **bump gesture handler**
- **tweak types**
- **add props back**
- **update types**
- **update other dialog views**
- **replace all buttons with `BottomSheetButton`**
- **fix type**
- **wizard remove buttons**
- **fix**
- **fix embed consent**
- **buttons in account switcher**
- **custom bottom sheet link**
- **nesting support**
- **android support**
- **fix type**
- **simplify pressable changes**
- **rm**
- **rename for clairity**
- **fix other buttons in dialogs**
- **tweak**
- **more fixes**

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ export type ButtonProps = Pick<
     style?: StyleProp<ViewStyle>
     hoverStyle?: StyleProp<ViewStyle>
     children: NonTextElements | ((context: ButtonContext) => NonTextElements)
+    PressableComponent?: React.ComponentType<PressableProps>
```
Comment (mozzius) [inline src/components/Button.tsx:90]: Extremely neat, we can stick `PressableScale` in here now presumably?

File: src/components/dialogs/PostInteractionSettingsDialog.tsx [hunk 2]
```diff
@@ -303,7 +304,8 @@ export function PostInteractionSettingsForm({
               }
               value={quotesEnabled}
               onChange={onChangeQuotesEnabled}
-              style={[, a.justify_between, a.pt_xs]}>
+              style={[, a.justify_between, a.pt_xs]}
```
Comment (mozzius) [inline src/components/dialogs/PostInteractionSettingsDialog.tsx:307]:
```suggestion
              style={[a.justify_between, a.pt_xs]}
```

File: src/view/com/util/post-ctrls/RepostButton.tsx [hunk 3]
```diff
@@ -157,7 +157,6 @@ let RepostButton = ({
             </View>
             <BottomSheetButton
               label={_(msg`Cancel quote post`)}
-              onAccessibilityEscape={close}
```
Comment (mozzius) [inline src/view/com/util/post-ctrls/RepostButton.tsx:160]: Is this necessary? (I never quite understood how this prop works)

**Review by mozzius — APPROVED** [review]
lgtm

File: src/view/com/util/post-ctrls/RepostButton.tsx (hunk 3 above)
Comment (haileyok) [reply src/view/com/util/post-ctrls/RepostButton.tsx:160]: It isn't supported on the RNGH pressable unfortunately (causes a crash). From what I read about it, I think the native dialog actually handles this for us (i think it lets the user use two finger presses to like dismiss or something...might look into the prop more later)

File: src/components/Button.tsx (hunk 1 above)
Comment (haileyok) [reply src/components/Button.tsx:90]: Yea that actually should work!

**Pull request merged**